next-project-directory/src/folder/ui-folder-info/FolderInfo.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from "./FolderInfo.module.scss";
+import classNames from "classnames/bind";
+
+const cx = classNames.bind(styles);
+
+type FolderInfoProps = {
+  profileImage: string;
```

Comment:
img의 src 속성에 넣어줄 값이라 이렇게 표현되면 좋겠네요!
```suggestion
  profileImageUrl: string;
```